### ISSUE TITLE
Minifies class name and function variables to be the same variable name

### ISSUE THREAD

**Issue by MarshallOfSound** [babel/babili#161]
Reproduction Code:

```js
class Foo {
  doThing() {
    
  }
}

thing.on('_', (something, anotherThing) => {
  const b = new Foo(anotherThing, something.sender);
}); 
```

Results in:

```js
class a{
  doThing(){}
}

thing.on('_', (a, c) => {
  new a(c, a.sender)
});
```

This can be seen [here](http://babeljs.io/repl/#?babili=true&evaluate=true&lineWrap=false&presets=react&code=class%20Foo%20%7B%0A%20%20doThing()%20%7B%0A%20%20%20%20%0A%20%20%7D%0A%7D%0A%0Athing.on('_'%2C%20(something%2C%20anotherThing)%20%3D%3E%20%7B%0A%20%20const%20b%20%3D%20new%20Foo(anotherThing%2C%20something.sender)%3B%0A%7D)%3B%20)

Basically `something` and `Foo` both minify to be `a` even though they are both accessible in the same scope.  I have no idea what is causing this but it is throwing all sorts of errors and issues in minified code 👍

**Comment by goto-bus-stop** [comment]
I think this is the same bug as #147, which is fixed on master:

```bash
➜  babili git:(master) ✗ cat test-161.js
class Foo {
  doThing() {

  }
}

thing.on('_', (something, anotherThing) => {
  const b = new Foo(anotherThing, something.sender);
});

➜  babili git:(master) ✗ ./packages/babili/bin/babili.js test-161.js
class Foo{doThing(){}}thing.on('_',(a,c)=>{new Foo(c,a.sender)});
```

**Comment by MarshallOfSound** [comment]
@goto-bus-stop Scanned the open issues, wasn't expecting it to be on the closed pile 👍 

I'll just check latest master and close if the issue is fixed 😄

**Comment by boopathi** [comment]
Thanks!. This is already fixed by #140 and is merged to master.

**Comment by MarshallOfSound** [comment]
@boopathi Is there any way we can get a patch bump for this, it's a pretty significant blocker at the moment
